contribution/translation/prevalidator.php

```diff
@@ -42,6 +44,23 @@ class prevalidator
 		'language/en/LICENSE',
 		'language/en/CHANGELOG',
 		'language/en/VERSION',
+
+		// Ignore markdown files too
```

Comment:
instead of adding explicit .md versions, why not just test for the possible existence of any extension to the file names listed above, so we can also catch, for example, .txt as well as .md

Reply:
I'm happy to make that change, but maybe we should restrict it purely to no extension, .md and .txt? That way we'll get a validation error if someone put garbage like `LICENSE.exe` or something.